src/config/config.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  server_url: 'https://private-anon-dcd46c5859-cristinagf.apiary-mock.com',
```

Comment:
Good idea, @davidecorreu @davidecorreu we should all run server in own computer and let that connect to db (in davids computer)